### DISCUSSION TITLE
Render the input as markdown, just like the output.

### DISCUSSION THREAD

**Discussion by GrayXu** [open-webui/open-webui#4271] [Ideas]
**Is your feature request related to a problem? Please describe.**
code blocks cannot be rendered correctly in the input.
I guess you might think that users won't input in MD format, or that rendering by default as MD would incorrectly render non-MD user input. But plain text actually doesn't tend to cause conflicts. ChatGPTNextWeb/ChatGPT-Next-Web also renders inputs as MD by default.

**Describe the solution you'd like**
Render the input as markdown, just like the output.

thanks

**Comment by atgehrhardt** [comment]
Would this just be for quality of life? Due to the underlying technology behind LLMs it wouldn’t make much of a difference for the accuracy of the input

**Reply by CookSleep** [reply]
The problem is I really do need to use code blocks as one of my code backup methods, in order to quickly copy old code to replace the incorrect new code output by LLM :(

**Reply by GrayXu** [reply]
Similar to @CookSleep's reasoning, this isn't specifically for optimizing LLM prompts (which typically only require simple delimiters).  
It's more about enhancing the readability of code blocks and improving the overall browsing experience.

**Comment by GrayXu** [comment]
It appears that this issue has been resolved in version 0.3.14, and gratitude goes to the developers for their contribution!
